CNAME

```diff
@@ -1 +0,0 @@
-www.srihash.org
```

Comment:
I guess you need this for testing, but I don't want this in the pull request :-))